### PR TITLE
Feature#40 clean project structure

### LINKED ISSUE #39: setting up for production
# REQUIREMENTS
Write a detailed description for this issue, provide all necessary information so any member of the project understands

# DESIGN
Provide references to images of the design, how it looks like now, and how it should look like.

# Acceptance Criteria
List all required acceptance criteria
- [ ] Criteria 1
- [ ] Criteria 2

### LINKED ISSUE #40: clean project structure
# REQUIREMENTS
- some links in our web app doesnt works or no way to access a view
-  Only show current user information
-  Only show current admin information
 - If it is admin, you can only manage the users, departments, places and areas of its institution
 - show an alert in places view if there is at least one positive person in the last 10 days.

# DESIGN 
No apply

# Acceptance Criteria
List all required acceptance criteria
- [ ] show the correct information
- [ ] all links redirect to your views

### PR DESCRIPTION
**Pull request by mariormt17**
# Description
 - If the user has the super admin role then he can do everything.
 - Only information related to the institution linked to the current user is showed.
 - The styles of the navbar and sidebar were added
 - Allows creating, editing and deleting data related to the current user's institution.
 - Create, display, edit and delete features were implemented in the user and role models. 
 - Added a tag in the places index to show how many active cases of covid-19 exist in each of them
 
 # Testing
 Create an user with super admin role and other user with another role and see the differences.
  
  ## Screenshots 
image
image

### CODE REVIEW

File: alerta-covid/app/controllers/application_controller.rb [hunk 1]
```diff
@@ -22,4 +22,8 @@ def current_ability
   def load_permissions
     @current_permissions = current_user.role.permissions.collect { |i| [i.subject_class, i.action] }
   end
+  
+  def current_institution
+    @current_institution ||= current_user.departament.institution if current_user
+  end
```
Comment (MiguelOcegueraM) [inline alerta-covid/app/controllers/application_controller.rb:28]: nice

**Review by MiguelOcegueraM — DISMISSED** [review]
good job mates, the code looks nice! too clean

**Review by FranckZnt19 — DISMISSED** [review]
A good job and very clean

**Review by FranckZnt19 — APPROVED** [review]

**Review by ArielMD — APPROVED** [review]

File: alerta-covid/app/javascript/packs/layouts/navbar.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+const DAYS = [
+  "Sunday",
+  "Monday",
+  "Tuesday",
+  "Wednesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+];
```
Comment (ramasoftdev) [inline alerta-covid/app/javascript/packs/layouts/navbar.js:12]: It's Ok, amazing. Only u have to remember single quotes instead double.

**Review by ramasoftdev — APPROVED** [review]
LGTM. Everything is OK.

**Pull request merged**